app/account/views.py

```diff
@@ -9,7 +9,7 @@
 from ..decorators import merchant_or_vendor_required
```

Comment:
both merchant vendor functionality is abstracted to the account class. no reason to repeat functionality that affects both twice in the vendor and merchant views.